engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/StartPlayingScreen.java

```diff
@@ -101,6 +102,16 @@ public void initialise() {
         WidgetUtil.trySubscribe(this, "mainMenu", button -> {
             getManager().pushScreen("engine:mainMenuScreen");
         });
+
+        WidgetUtil.trySubscribe(this, "renderingSettings", button -> {
```

Comment:
I don't know enough about the menu system to work out the actual reason quickly, but returning from the render module set up screen then returning again (from world preview to pregeneration) doesn't work correctly (it goes back to the render module setup instead).